tgui/packages/tgui/interfaces/Pacman.js

```diff
@@ -76,7 +76,7 @@ export const Pacman = (props, context) => {
                       <NumberInput
                         value={output_set}
                         minValue={1}
-                        maxValue={output_max}
+                        maxValue={Infinity}
```

Comment:
why not have the max value set to `max value * 2.5`?  Players might think they can go higher than that

Reply:
done